modules/distribution/src/assembly/bin.xml

```diff
@@ -1007,16 +1007,6 @@
             </source>
             <outputDirectory>wso2is-${pom.version}/repository/conf</outputDirectory>
         </file>
-
-        <file>
-            <source>
-                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/identity/service-providers/sp_dashboard.xml
```

Comment:
We should remove the `sp_dashboard.xml` from the original feature as well.

Reply:
The feature is removed so no longer we have sp_dashbard.xml in the pack.